src/main/java/com/ldtteam/blockout/views/ZoomDragView.java

```diff
@@ -42,12 +43,12 @@ public ZoomDragView()
     public ZoomDragView(final PaneParams params)
     {
         super(params);
-        dragFactor = params.getDoubleAttribute("dragfactor", dragFactor);
-        dragEnabled = params.getBooleanAttribute("dragenabled", dragEnabled);
-        zoomFactor = params.getDoubleAttribute("zoomfactor", zoomFactor);
-        zoomEnabled = params.getBooleanAttribute("zoomenabled", zoomEnabled);
-        minScale = params.getDoubleAttribute("minscale", minScale);
-        maxScale = params.getDoubleAttribute("maxscale", maxScale);
+        dragFactor = params.getDoubleAttribute("dragFactor", dragFactor);
```

Comment:
Maybe (probably) its just me, but I expect all markup attributes to be in lowercase or kebab-case. To me this is less guessable. I know the standard does have them, but I haven't really seen it round too often, bar the schema one.

Reply:
yeah, the others are maxscale so this should be dragfactor

Reply:
kk, gonna change the schema